corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1546,9 +1546,39 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
         events: {
             'click #app-main': 'onClickAppMain',
         },
+        handleSmallScreenChange: function (smallScreenEnabled) {
```

Comment:
I'd love to be able to approach this with something like this:
```html
<div id="persistent-menu-content" style="display: none;">
  ...
</div>
<div id="persistent-menu-small">
  <button data-bs-toggle="offcanvas">...</button>
  <div class="offcanvas">
    {{ persistent-menu-content }}
  </div>
</div>
<div id="persistent-menu-big">
  <button data-bs-toggle="collapse">...</button>
  <div class="collapse show">
    {{ persistent-menu-content }}
  </div>
</div>
```
Where the containing context is swapped depending on which screen size is active, or perhaps the same DOM is displayed in two places, and we use media queries to figure out which to hide and which to show.  I don't know a good way to do that though.  Had the same problem with the login-as banner/pill.